config/packages/cache.yaml

```diff
@@ -15,5 +15,9 @@ framework:
         #app: cache.adapter.apcu
 
         # Namespaced pools use the above "app" backend by default
-        #pools:
-            #my.dedicated.cache: ~
+
+        pools:
+            # This pool should be used only by SiteFactory bundle
+            site_factory_pool:
```

Comment:
any reason we're forcing this on Integrators? Usually when they switch to Redis, they have a reason to.